langs.go

```diff
@@ -92,3 +92,13 @@ func IsValid(lang string) bool {
 func NativeName(lang Lang) string {
 	return lang.Native
 }
+
+// Parse returns the Lang struct for a given language.
```

Comment:
El tipo que tiene por debajo, más después de lo que has hecho, es algo privado de este paquete que no queremos exponer.

```suggestion
// Parse returns the Lang for a given language.
```